parser/sqlparser/options_utils.go

```diff
@@ -0,0 +1,37 @@
+package sqlparser
+
+func injectEntry(v interface{}, entry OptionsSpecsEntry) interface{} {
```

Comment:
Please add a comment describing what this function does (I'm not sure I understand it)